game-core/src/test/java/games/strategy/triplea/attachments/TerritoryAttachmentTest.java

```diff
@@ -0,0 +1,76 @@
+package games.strategy.triplea.attachments;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.hasSize;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import games.strategy.engine.data.GameData;
+import games.strategy.engine.data.PlayerID;
+import nl.jqno.equalsverifier.EqualsVerifier;
+
+final class TerritoryAttachmentTest {
+  @Nested
+  final class GetCaptureOwnershipChangesTest {
+    private final GameData gameData = new GameData();
+    private final PlayerID player1 = new PlayerID("player1", gameData);
+    private final PlayerID player2 = new PlayerID("player2", gameData);
+    private final PlayerID player3 = new PlayerID("player3", gameData);
+    private final TerritoryAttachment territoryAttachment =
+        new TerritoryAttachment("territoryAttachment", null, gameData);
+
+    private String join(final String... values) {
```

Comment:
This method could be made static

Reply:
Unfortunately not.  Static methods are not permitted on inner classes (I never really understood the reasoning behind that).